pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -99,6 +100,20 @@ async def acker(subscription: str,
                 log.warning(
                     'Ack request failed, better luck next batch', exc_info=e)
                 metrics_client.increment('pubsub.acker.batch.failed')
+
+                is_response_error = isinstance(
+                    e,
+                    aiohttp.client_exceptions.ClientResponseError)
+                if is_response_error and e.status == 400:  # type: ignore # pylint: disable=no-member
```

Comment:
Any other HTTP error code relevant? Other `4xx` class errors maybe (although some are transient in GCP, like occasional 404s when network is bad)?

Reply:
Invalid `ack_id` also throws 400, and I'm out of other ideas :D

Reply:
Great, 400 works 😛